### ISSUE TITLE
"DAV: calendar-access" and "DAV: addressbook" not sent

### ISSUE THREAD

**Issue by naman-dixit** [ckulka/baikal-docker#19]
As I reported [here](https://github.com/sabre-io/Baikal/issues/838) and [here](https://forums.bitfire.at/topic/2148/baikal-doesn-t-provide-required-carddav-service/7), the `baikal:nginx` image seems to have troubles with responding to OPTIONS requests from DAVx5.

**Comment by ckulka** [comment]
Hi @naman-dixit ,

sorry for the delay - this looks like an unintended side-effect of #13 where Nginx received additional CORS behavior: all `OPTIONS` requests return no content instead of being being passed to Baikal (I simply copied that part from https://enable-cors.org/server_nginx.html).

https://github.com/ckulka/baikal-docker/blob/bcb92d8b0fde7a07154d53b75560a63a0846d852/files/nginx.conf#L29-L82

I'll commit a change to pass option requests to the backend too and write once it's available in the experimental build, which then hopefully works with DAVx5.

**Comment by barskern** [comment]
I just wanted to add that I had the same problem. I will try out the experimental version as mentionned.

**Comment by ckulka** [comment]
I'll close this together with #20, i.e. the release of version 0.6.1

**Comment by ckulka** [comment]
The configuration update has been published together with 0.6.1 from #20

**Comment by richy-notime** [comment]
It seems like the issue is still not fixed.
Setting up a docker container with:
```
/usr/bin/docker run \
          --rm --name baikal \
          -v /daemons/baikal:/var/www/baikal/Specific \
          -p 1825:80 \
          ckulka/baikal:nginx
```
Configurating the admin account, setting up a user account and some calendars and addressbooks.
But when i try to connect with DAVx5 using the pattern:
```
base-url: http://myserverip:1825/dav.php/
```
nothing is found. Different Clients although have no sync problems (kmail, Android: CardDAV-Sync free) .
